openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/Component/ProfilerSettingsModal.tsx

```diff
@@ -626,64 +624,236 @@ const ProfilerSettingsModal: React.FC<ProfilerSettingsModalProps> = ({
                   />
                 </Form.Item>
               </Col>
-              <Col span={12}>
-                <Form.Item
-                  className="m-b-0"
-                  label={<span className="text-xs">{t('label.interval')}</span>}
-                  labelCol={{
-                    style: {
-                      paddingBottom: 8,
-                    },
-                  }}
-                  name="partitionInterval"
-                  rules={[
-                    {
-                      required: state?.enablePartition,
-                      message: t('message.field-text-is-required', {
-                        fieldText: t('label.interval'),
-                      }),
-                    },
-                  ]}>
-                  <InputNumber
-                    className="w-full"
-                    data-testid="interval-required"
-                    disabled={!state?.enablePartition}
-                    placeholder={t('message.enter-interval')}
-                    size="middle"
-                  />
-                </Form.Item>
-              </Col>
               <Col span={12}>
                 <Form.Item
                   className="m-b-0"
                   label={
-                    <span className="text-xs">{t('label.interval-unit')}</span>
+                    <span className="text-xs">
+                      {t('label.column-entity', {
+                        entity: t('label.name'),
+                      })}
+                    </span>
```

Comment:
If this is what style we want to apply on every form label can we do it globally?

Reply:
this is not the usual style, but it is particularly related to this form, to show the subsection